samples/openthread/coap_client/src/coap_client.c

```diff
@@ -133,6 +137,20 @@ void main(void)
 		power_down_unused_ram();
 	}
 
+#ifdef CONFIG_BOOTLOADER_MCUBOOT
+	/* Check if the image is run in the REVERT mode and eventually
+	 * confirm it to prevent reverting on the next boot.
+	 */
+	if (mcuboot_swap_type() == BOOT_SWAP_TYPE_REVERT) {
+		if (boot_write_img_confirmed()) {
```

Comment:
this call confirm only not already confirmed image, if is not needed (but do no harm).

Reply:
I agree, but my purpose was to always confirm the image here in order to simplify example and reduce necessary interaction with the user.

Reply:
ok, others do the same.